doc/classes/AnimationNodeSub2.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="AnimationNodeSub2" inherits="AnimationNodeSync" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Blends two animations subtractively inside of an [AnimationNodeBlendTree].
+	</brief_description>
+	<description>
+		A resource to add to an [AnimationNodeBlendTree]. Blends two animations subtractively based on an amount value.
+		This is basically used for pre-calculation to cancel the extra poses from the animation to be used for [AnimationNodeAdd2] / [AnimationNodeAdd3] to extract the delta animation.
+		In general, the blend value should be in the [code][0.0, 1.0][/code] range, but you can use outside of that range for blending amplified or inverted animations.
+		[b]Note:[/b] The behavior is different from using a negative value of amount in [AnimationNodeAdd2] since matrix (transform) calculations do not have commutative law. [AnimationNodeSub2] multiplies the inverted animation from the left side, but negative [AnimationNodeAdd2] multiplies the inverted animation from the right side.
```

Comment:
```suggestion
		A resource to add to an [AnimationNodeBlendTree]. Blends two animations subtractively based on a numerical value.
		This animation node is usually used for pre-calculation to cancel out any extra poses from the animation. The output can then be used to extract the delta animation with [AnimationNodeAdd2] or [AnimationNodeAdd3].
		In general, the blend value should be in the [code][0.0, 1.0][/code] range, but values outside of this range can be used for amplified or inverted animations.
		[b]Note:[/b] This animation behavior is different from using a negative value in [AnimationNodeAdd2], since the transformation matrices do not satisfy the commutative law. [AnimationNodeSub2] multiplies the transformation matrix of the inverted animation from the left side, while negative [AnimationNodeAdd2] multiplies it from the right side.
```

Reply:
This is not meant to get the delta animation of the Add2/Add3 result, but rather to use it as input to the "add" port of Add2/Add3, so I made the following change:
`This animation node is usually used for pre-calculation to cancel out any extra poses from the animation for the "add" animation source in [AnimationNodeAdd2] or [AnimationNodeAdd3].`